sql/mob_skills.sql

```diff
@@ -1364,7 +1364,7 @@ INSERT INTO `mob_skills` VALUES (1527,1086,'laser_shower',4,10.0,2000,1500,4,0,0
 INSERT INTO `mob_skills` VALUES (1528,1087,'floodlight',2,15.0,2000,1500,4,0,0,0,0,0,0);
 INSERT INTO `mob_skills` VALUES (1529,1089,'hyper_pulse',1,10.0,2000,1500,4,0,0,0,0,0,0); -- Proto-Omega
 INSERT INTO `mob_skills` VALUES (1530,1088,'stun_cannon',4,20.0,2000,1500,4,0,0,0,0,0,0);
-INSERT INTO `mob_skills` VALUES (1531,772,'wz_recover_all',1,20.0,3000,0,4,0,0,0,0,0,0);
+INSERT INTO `mob_skills` VALUES (1531,772,'wz_recover_all',1,20.0,0,0,4,0,0,0,0,0,0);
```

Comment:
What is this change?

Reply:
Limbus recover Armoury Crates are mobs on the mob allegiance and that skill, `wz_recover_all`, targets the players and heals them. I noticed there was a huge delay and looked on retail and noticed that it shouldn't have any delay. The reason I included it here is because in order for recover Armoury Crates to target the player they have to be in the same battle ID which means that technically they can be hit by AoE spells when activated. I reduced the delay in hopes to reduce the likelihood of this happening.

There is quite possibly a better solution. One thought is to add a special targeting type on skills that ignore battleID. Another would be to add a new flag that says the entity cannot be targeted at all.

Reply:
I'd go with editing the target flags to have one for it, but there might be a flag already to prevent targetting. I wanna say there is and I'd seen it used for that one abyssea NM (big clot type slime) but I am unsure what it was enum'd as to direct you to it.

Reply:
actually iirc that flag was bugged and ppl coudl cheat with aoes anyway..fixing it may be the answer